pkg/vm/engine/tae/catalog/model.go

```diff
@@ -147,6 +160,16 @@ func init() {
 		Oid:  types.T_timestamp,
 		Size: 8,
 	}
+	tvarchar := types.Type{
+		Oid:   types.T_varchar,
+		Size:  24,
+		Width: 100,
+	}
+	tsinglechar := types.Type{
```

Comment:
This is not correct.   T_char type is also veriable length.

In fact, I suggest we change all these types to tu32 := types.T_uint32.ToType(),    tsinglechar = types.T_char.ToType(), etc.

Reply:
yeah, using ToType() makes it clearer